calypso/service.go

```diff
@@ -43,7 +43,7 @@ import (
 	"go.dedis.ch/cothority/v3/skipchain"
 	"go.dedis.ch/kyber/v3"
 	"go.dedis.ch/kyber/v3/share"
-	"go.dedis.ch/kyber/v3/share/dkg/pedersen"
+	dkg "go.dedis.ch/kyber/v3/share/dkg/pedersen"
```

Comment:
Makes me smile: between my IntelliJ and your emacs, one deems that the `dkg` is necessary, while the other wants to remove it...